test/e2e/framework/test_context.go

```diff
@@ -296,12 +300,61 @@ func ViperizeFlags() {
 	AfterReadingAllFlags(&TestContext)
 }
 
+func createKubeConfig(clientCfg *restclient.Config) *clientcmdapi.Config {
+	clusterNick := "cluster"
```

Comment:
@sttts Are there any utilities for converting an in-cluster-config to an on-disk kubeconfig?  

It seems weird that there isn't a utility to do this.

Reply:
unfortunately did not find any ... i picked this from a test case - https://github.com/kubernetes/kubernetes/blob/master/test/integration/examples/apiserver_test.go#L363